modules/core/src/test/java/org/apache/ignite/internal/processors/cache/IgniteOnePhaseCommitNearReadersTest.java

```diff
@@ -97,13 +92,11 @@ private void putReadersUpdate(int backups) throws Exception {
 
         awaitPartitionMapExchange();
 
-        client = true;
-
         Ignite srv = ignite(0);
 
         srv.createCache(cacheConfiguration(backups));
 
-        Ignite client1 = startGrid(SRVS);
+        Ignite client1 = startClientGrid(SRVS);
```

Comment:
The following `Ignite client2 = startGrid(SRVS + 1);` also starts a client node.

Reply:
Fixed.